packages/rocketchat-autotranslate/server/autotranslate.js

```diff
@@ -122,6 +141,11 @@ export class AutoTranslate {
 		message.html = message.msg;
 		message = RocketChat.Markdown.parseMessageNotEscaped(message);
 		message.msg = message.html;
+		// Some parsers (e. g. Marked) wrap the complete message in a <p> - this is unnecessary and should be ignored with respect to translations
+		const wrappedInParagraph = message.msg.startsWith('<p>') && message.msg.endsWith('</p>');
```

Comment:
Hope its not just ```<p>``` but also other tags. For example ```<strong>```.

Reply:
I only stripped the outermost `<p>` - this shouldn't make a problem, I hope